deps/npm/node_modules/normalize-package-data/node_modules/is-builtin-module/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+var builtinModules = require('builtin-modules');
```

Comment:
I don't think this will be reliable in the future.

@zkat can you open an issue about that we should expose this list properly?